css/style.css

```diff
@@ -1,3 +1,59 @@
+body {
+  background-color: #ccc;
+  background-size: cover;
```

Comment:
if you add "background-repeat: none" and "background-align: center" rules you can avoid the background tiling when you add the different images in there 👍